pkg/mount/mountinfo.go

```diff
@@ -1,7 +1,38 @@
 package mount
 
+// Reveals information about a particular mounted filesystem. This struct is
```

Comment:
Docstring should start from documenting entity name. For example:
```
MountInfo represents...
```
Also, that problem with other docstrings as well. You can find examples of docstrings in standard library.